tests/unit_test/llm_test.py

```
@@ -1059,19 +1460,13 @@ async def test_gpt3_faq_embedding_predict_exact_match(self, mock_embedding, mock
-        assert not DeepDiff(mock_embedding.call_args[1], expected, ignore_order=True)
-
     @pytest.mark.asyncio
-    @mock.patch.object(litellm, "aembedding", autospec=True)
-    async def test_gpt3_faq_embedding_predict_embedding_connection_error(self, mock_embedding):
-        embedding = list(np.random.random(LLMProcessor.__embedding__))
+    @mock.patch.object(LLMProcessor, "get_embedding", autospec=True)
+    async def test_gpt3_faq_embedding_predict_embedding_connection_error(self, mock_get_embedding, aioresponses):
         user = "test"
         bot = "test_gpt3_faq_embedding_predict_embedding_connection_error"
         key = "test"
+        llm_type = "openai"
```

Ruff (0.8.2)

Local variable `llm_type` is assigned to but never used

Remove assignment to unused variable `llm_type`

(F841)